src/fixtures/users.js

```diff
@@ -0,0 +1,32 @@
+export default [
+  {
+    id: 1,
+    email: "test@test.com.br",
+    created_at: "2015-01-26T23:19:00.575Z",
+    updated_at: "2016-08-22T09:23:23.947Z",
+    first_name: "Matt",
+    last_name: "Tester",
+    display_email: null,
+    slug: "matt-tester",
+    youtube_id: null,
+    display_profile: true,
+    latitude: 27.5949,
+    longitude: 48.5482,
+    country_name: "Brazil",
+    city: "Florianopolis",
+    region: "Santa Catarina",
+    youtube_user_name: null,
+    github_profile_url: null,
+    display_hire_me: null,
+    bio: null,
+    receive_mailings: true,
+    timezone_offset: -18000,
+    country_code: "BR",
+    status_count: 0,
+    deleted_at: null,
+    event_participation_count: 0,
+    can_see_dashboard: false,
+    skill_list: [ ],
+    title_list: [ ]
+    },
+]
```

Comment:
new line

Reply:
add a new line? 
will we set up linting?

Reply:
yes